Sources/Net/NetSocket.CircularBuffer.swift

```diff
@@ -45,7 +45,8 @@ extension NetSocket {
                 self.locked = locked
             }
             let length = min(count, capacity - tail)
-            return self.data.withUnsafeMutableBytes { (bytes: UnsafeMutableRawBufferPointer) -> Bool in
```

Comment:
I have a concern about performance issue.
I will change a struct to class.

Reply:
So I will reject this issue.

Reply:
I believe it would be the same, isn't it a copy on write? managed by the compiler?  So I think the previous (current) version was the same (still a copy on write), just unsafe. Here; it's expliit.   But: I havn't profiled this.